src/components/User/Sales/components/SaleList.vue

```diff
@@ -223,23 +218,27 @@ export default {
 
   watch: {
     'filters.openOnly' () {
-      this.getList()
+      this.$refs.collectionLoaderBtn.loadFirstPage()
     },
     'owner': _.throttle(function () {
       this.getOwner()
-      this.getList()
+      this.$refs.collectionLoaderBtn.loadFirstPage()
     }, 1000),
     'filters.name': _.throttle(function () {
-      this.getList()
+      this.$refs.collectionLoaderBtn.loadFirstPage()
     }, 1000),
     'filters.baseAsset': _.throttle(function () {
-      this.getList()
+      this.$refs.collectionLoaderBtn.loadFirstPage()
```

Comment:
since you reuse such unclear executing of loadFirstPage, I’d prefer you to wrap it in a method like `reloadCollecionLoader`. naming is up to you. Also please rechek all the other occurences of such issue